src/core/parentClient.ts

```diff
@@ -88,4 +88,29 @@ export class ParentClient extends BaseClient {
     }
     throw new Error("No pupil with specified ID returned");
   }
+    /**
+   * Changes the login password for this parent's account
+   * @param current Current password
+   * @param new New password
+   * @returns whether it was successful or not
+   */
+    async changePassword(this: ParentClient, current: string, newPassword: string): Promise<ChangePasswordResponse>{
```

Comment:
```suggestion
    async changePassword(currentPassword: string, newPassword: string): Promise<ChangePasswordResponse> {
```
Unsure why `this` is required. And current should be renamed to `currentPassword` for consistency